docs/integrations.md

```diff
@@ -156,3 +156,25 @@ In addition, ruff publishes the following images:
 
 As with the distroless image, each image is published with ruff version tags as
 `ruff:{major}.{minor}.{patch}-{base}` and `ruff:{major}.{minor}-{base}`, e.g., `ruff:0.6.6-alpine`.
+
+## GitLab CI/CD
+
+You can add the following configuration to `.gitlab-ci.yml` to run a `ruff format`, and a `ruff check` compatible with GitLab's codequality.
+
+```yaml
+Ruff:
+  stage: .pre
+  interruptible: true
+  image:
+    name: ghcr.io/astral-sh/ruff:0.7.1-alpine
+  variables:
+    CODE_LOCATION: $CI_PROJECT_DIR
+  script:
+    - cd $CODE_LOCATION
+    - ruff --version
+    - ruff check --output-format=gitlab > code-quality-report.json
+    - ruff format --diff --verbose
```

Comment:
Why the verbose flag on `format`?

Reply:
Ah yeah that's probably not necessary indeed, I do like to see which configs are being taken into account if I'm puzzled by a fail
```suggestion
    - ruff format --diff
```